docs/contributing-integrations.rst

```diff
@@ -114,7 +114,7 @@ Many of the tests are based on "snapshots": saved copies of actual traces sent t
 
 1. Update the library and test code to generate new traces.
 2. Delete the snapshot file corresponding to your test at ``tests/snapshots/<snapshot_file>`` (if applicable).
-3. Use `docker-compose up -d testagent` to start the APM test agent, and then re-run the test. Use `--pass-env` as described
+3. Use `docker compose up -d testagent` to start the APM test agent, and then re-run the test. Use `--pass-env` as described
```

Comment:
Makes total sense.  I agree that it's most sensible for documentation and internal guidance to use the newer, shinier `docker compose` instead of the old and crusty `docker-compose`.